lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1181,16 +1181,20 @@ natural = Quantity . fromIntegral . getQuantity
 
 getWalletWithCreationTime
     :: forall s t k w. ()
-    => (WalletId -> Wallet s -> WalletMetadata -> SyncProgress -> Set Tx -> w)
+    => (WalletId -> Wallet s -> WalletMetadata -> SyncProgress -> Set Tx -> (Quantity "lovelace" Word64) -> w)
     -> ApiLayer s t k
     -> ApiT WalletId
     -> Handler (w, UTCTime)
 getWalletWithCreationTime mk ctx (ApiT wid) = do
-    (wallet, meta, pending) <-
+    (wallet, meta, pending, rewards) <-
         liftHandler $ withWorkerCtx ctx wid throwE $
-            \wrk -> W.readWallet wrk wid
+            \wrk -> do
+                (a,b,c) <- W.readWallet wrk wid
+                r <- withExceptT (error "TODO") $ W.fetchWalletRewardBalance wrk wid
```

Comment:
<s>This doesn't compile when I require `HasNetworkLayer t ctx` from `W.fetchWalletRewardBalance `</s> 🤔 

Ok, explicit type application helped.